src/app/modules/group/pages/group-access/group-access.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { Group } from '../../http-services/get-group-by-id.service';
+import { GetItemByIdService } from '../../../item/http-services/get-item-by-id.service';
+import { ReplaySubject, of } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { mapToFetchState } from '../../../../shared/operators/state';
+
+@Component({
+  selector: 'alg-group-access',
+  templateUrl: './group-access.component.html',
+  styleUrls: [ './group-access.component.scss' ],
+})
+export class GroupAccessComponent implements OnChanges {
+  @Input() group?: Group;
+
+  private readonly rootActivityId$ = new ReplaySubject<string | null>(1);
+
+  itemState$ = this.rootActivityId$.pipe(
```

Comment:
`itemState` as variable name is a bit strange. Either use `state` as at many places to consider that it is the state of component globally. Or, if we consider we may have multiple states later in this component, that's more `rootActivityState$`. `itemState` does not help understanding what it is for, it is just define the type which may be in the state.

Reply:
Yes, I'm expecting multiple states, but `rootActivityState$` more clear. I'll fix

Reply:
Fixed